src/smt/smt_setup.cpp

```diff
@@ -129,6 +129,8 @@ namespace smt {
             setup_QF_FPBV();
         else if (m_logic == "QF_S" || m_logic == "QF_SLIA")
             setup_QF_S();
+        else if (m_logic == "QF_CHAR")
```

Comment:
Can you revert changes in smt_setup.h and smt_setup.cpp?
It would introduce a path towards QF_CHAR support for an ad-hoc logic, which I would like to avoid.